README.md

```diff
@@ -95,6 +95,9 @@ Note: Recent versions of Bismark automatically generate a BAM file instead of a
  as described in the following section.
 
 ## Duplicate Determination with NuDup (Optional):
+
+Follow instructions to setup NuDup here: https://github.com/nugentechnologies/nudup
```

Comment:
The line here seems out-of place. The section may need a bit of additional re-writing.